tests/QuicksandDeleteTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+use Carbon\Carbon;
+use Illuminate\Config\Repository;
+use Illuminate\Database\Capsule\Manager as DB;
+use Tightenco\Quicksand\DeleteOldSoftDeletes;
+
+// @todo move this somewhere
+class Person extends \Illuminate\Database\Eloquent\Model
+{
+    use \Illuminate\Database\Eloquent\SoftDeletes;
+
+    protected $fillable = ['name'];
+}
+
+class QuicksandDeleteTest extends FunctionalTestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+        $this->migrate();
+    }
+
+    public function migrate()
+    {
+        $this->migratePeopleTable();
+    }
+
+    public function migratePeopleTable()
+    {
+        DB::schema()->create('people', function ($table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->timestamps();
+            $table->softDeletes();
+        });
+    }
+
+    public function act()
+    {
+        $configSpy = Mockery::spy(Repository::class);
```

Comment:
Feels weird to have assertions in act. To me, `act()` is just saying` do the thing. I'm guessing we have them in here because this is necessary for preparing the spy to work right?